ctapipe/core/container.py

```diff
@@ -18,33 +24,111 @@ class Field:
         Help text associated with the item
     unit: `astropy.units.Quantity`
```

Comment:
unit should not be a quantity

Reply:
good point.What's a bit annoying is that:
```py3
unit = u.m
isinstance(unit, u.Unit) # → False
type(unit) # →  astropy.units.core.IrreducibleUnit
```
I'm not sure the best way to specify the type of an astropy  unit... Perhaps `Union[u.core.CompoundUnit, u.core.IrriducibleUnit, str]`, but that's a bit ugly

Reply:
it's `u.UnitBase`, see: https://github.com/astropy/astropy/issues/6517

Reply:
I guess `u.core.UnitBase` works

Reply:
Eventually would be nice to use something like `mypy`, but that's going to need a lot of work since these types hints are a bit lazy